BUILD.md

```diff
@@ -0,0 +1,92 @@
+# Build PySlurm (20.11.2)
+
+## Prepare Slurm container (optional)
+
+```bash
+git clone https://github.com/bikerdanny/docker-centos-slurm
+cd docker-centos-slurm
+pip3 install j2cli
+j2 Dockerfile.j2 config.ini > Dockerfile
+docker build -t bikerdanny/slurm:20.02.5 .
+sed -i "s/controller1/c10/g" config.ini
+sed -i "s/compute1/c10/g" config.ini
+j2 aio.docker-compose.yml.j2 config.ini > docker-compose.yml
+mkdir var_lib_mysql
+mkdir etc_munge
+mkdir etc_slurm
+mkdir shared
+docker-compose up -d
+docker-compose exec c10 bash
+supervisorctl stop slurmctld
+echo "SwitchName=s2 Nodes=c10" > /etc/slurm/topology.conf
+echo "#
+# TOPOLOGY
+TopologyPlugin=topology/tree
+#
+# PlugStackConfig
+PlugStackConfig=/etc/slurm/plugstack.conf" >> /etc/slurm/slurm.conf
+sacctmgr show cluster # if the command returns successfully, you can start slurmctld again
+supervisorctl start slurmctld
+sinfo
+```
+The following steps you could do inside the container.
+
+## Checkout PySlurm
+
+```bash
+cd /root
+git clone https://github.com/bikerdanny/pyslurm.git
+cd pyslurm
+git checkout 20.02.5
```

Comment:
`20.02.5`?

Reply:
this has to be adjusted it is not an official release so I can adjust it